lib/idris-controller.coffee

```diff
@@ -46,6 +47,9 @@ class IdrisController
       @messages.attach()
       @messages.hide()
 
+  stopCompiler: =>
+    @model?.stop()
```

Comment:
The current alerts are very dire. To confirm a successful user action, we would want it not to appear like something went wrong.

Do we report the process state in the status bar? (I'd check directly, but on my phone just now.) Being able to inspect that and see that it is now dead might be all the confirmation needed.